vipps-checkout-api.md

```diff
@@ -132,6 +133,34 @@ sequenceDiagram
 
 ![porterbuddy_example](resources/porterbuddy_example.png)
 
+#### Instabox integration
+
+Instabox is a shipping provider that offers among others delivery to a locker location selected by the customer. To enable it, the merchant must:
```

Comment:
Can remove "among others". "offers delivery to a locker location" does not exclude other options.

Reply:
👍

Reply:
Like this